charts/matrix-stack/source/matrix-rtc.yaml.j2

```diff
@@ -18,7 +18,7 @@ restrictRoomCreationToLocalUsers: false
 # livekitAuth:
   {{- sub_schema_values.credential("The keys.yaml file for the LiveKit SFU\n## This is required if `sfu.enabled` is set to `false`", "keysYaml", initIfAbsent=False, commented=True) | indent(2) }}
   ## Provide a key and secret if not using an existing keys.yaml
-  # key: ""
+  # key: "matrix-rtc"
```

Comment:
This either needs to be uncommented or the `| default "matrix-rtc"` added back to both envs